docs/internals/ingest-v2.md

```diff
@@ -1,12 +1,18 @@
 # Ingest V2
 
-Ingest V2 is a new ingestion API that is designed to be more efficient and scalable for thousands of indexes than the previous version. It is currently in beta and is not yet enabled by default.
+Ingest V2 is the latest ingestion API that is designed to be more efficient and scalable for thousands of indexes than the previous version. It is the default since 0.9.
 
-## Enabling Ingest V2
+## Architecture
 
-To enable Ingest V2, you need to set the `QW_ENABLE_INGEST_V2` environment variable to `1` on the indexer, control-plane, and metastore services.
+Just like ingest V1, the new ingest uses [`mrecordlog`](https://github.com/quickwit-oss/mrecordlog) to persist ingested documents that are waiting to be indexed. But unlike V1, which always persists the documents locally on the node that receives them, ingest V2 can dynamically distribute them into WAL units called _shards_. The assigned shard can be local or on another indexer. The control plane is in charge of distributing the shards to balance the indexing work as well as possible across all indexer nodes. The progress within each shard is not tracked as an index metadata checkpoint anymore but in a dedicated metastore `shards` table.
 
-You also have to activate the `enable_cooperative_indexing` option in the indexer configuration. The indexer configuration is in the node configuration:
+In the future, the shard based ingest will also be capable of writing a replica for each shard, thus ensuring a high durability of the documents that are waiting to be indexed (durability of the indexed documents is guarantied by the object store).
+
+## Toggling between ingest V1 and V2
+
+Variables driving the ingest configuration are documented [here](../ingest-data/ingest-api.md#ingest-api-versions).
+
+With ingest V2, you can also activate the `enable_cooperative_indexing` option in the indexer configuration. The indexer configuration is in the node configuration:
```

Comment:
Can you add that cooperative indexing is useful for deployments handling dozens of indexes.